RFS/src/main/java/com/rfs/common/GlobalMetadata.java

```diff
@@ -14,6 +14,8 @@
 
 
 public class GlobalMetadata {
+    private GlobalMetadata() {}
```

Comment:
Note; in my outstanding change, these classes are promoted to 'functional', I think the rules are triggering here because there are no instance references so its seen as a utility class.

Recommendations like this are more about a code smell why does this class exist if it doesn't have any properties/fields - for example this could have been a package path